helpers/user.py

```diff
@@ -0,0 +1,13 @@
+
+import yaml
+from os.path import join, dirname
+from logger import pp
+
+stream = file(join(dirname(__file__), '..', 'config/user-map.yml'), 'r')
```

Comment:
install of using a manually maintained github_id-slack_id map, add a command to enable users to link their slack account and github account

Reply:
TODO: add the slack bot subscribe command, so that users could choose which notifications to subscribe, like reviewer added,  pr approve, review comment added etc